include/psp2/power.h

```diff
@@ -98,13 +98,41 @@ int scePowerGetBatteryRemainCapacity(void);
 SceBool scePowerIsLowBattery(void);
 
 /**
- * Returns battery full capacity ?
+ * Returns battery full capacity
  *
- * @return ?
+ * @return battery full capacity in mAh (milliampere hour)
  */
 int scePowerGetBatteryFullCapacity(void);
 
 /**
+ * Returns battery temperature
+ *
+ * @return temperature in degrees celcius * 100
+ */
+int scePowerGetBatteryTemp(void);
+
+/**
+ * Returns battery voltage
+ *
+ * @return battery voltage in kilo volts
```

Comment:
My bad, you're correct. I had it the wrong way round. 
Fixed with https://github.com/vitasdk/vita-headers/pull/168/commits/779a39336345281208280e5c68220979787863a1